docs/Annex.md

```diff
@@ -19,6 +21,8 @@ sudo reboot now
 sudo su montagu
 docker start barman-montagu
 docker exec barman-montagu setup-barman --no-initial-backup
+cd montagu-db-backup && ./start-metrics.sh
```

Comment:
There is a ticket to make these come back up automatically:
https://vimc.myjetbrains.com/youtrack/issue/VIMC-2299